utils.py

```diff
@@ -841,6 +841,36 @@ def compute_list_difference(list_a, list_b):
     return list(set(list_a) - set(list_b))
 
 
+def is_appengine_development_environment():
+    """Returns wheter the app is being run locally in a development server.
+    More information can be found here:
+    https://cloud.google.com/appengine/docs/standard/python/tools/
+    using-local-server#detecting_application_runtime_environment
+
+    Returns:
+        bool. Whether the current instance is running locally on a developer's
+        computer.
+    """
+    return ('APPENGINE_RUNTIME' in os.environ and
+            'Development/' in os.environ['SERVER_SOFTWARE'])
+
+
+def is_appengine_production_environment():
```

Comment:
Maybe change "production" to "cloud" because "production" is a bit overloaded (there is "local prod mode" too).

Also, missing test coverage?